content/github/site-policy/dmca-takedown-policy.md

```diff
@@ -91,17 +91,17 @@ When GitHub processes a DMCA takedown under our circumvention technology claim r
 
 We recognize that there are many valid reasons that you may not be able to make changes within the window of approximately 1 business day we provide before your repository gets disabled. Maybe our message got flagged as spam, maybe you were on vacation, maybe you don't check that email account regularly, or maybe you were just busy. We get it. If you respond to let us know that you would have liked to make the changes, but somehow missed the first opportunity, we will re-enable the repository one additional time for approximately 1 business day to allow you to make the changes. Again, you must notify us that you have made the changes in order to keep the repository enabled after that window of approximately 1 business day, as noted above in [Step A.4](#a-how-does-this-actually-work). Please note that we will only provide this one additional chance.
 
-### E. Transparency
+### D. Transparency
 
 We believe that transparency is a virtue. The public should know what content is being removed from GitHub and why. An informed public can notice and surface potential issues that would otherwise go unnoticed in an opaque system. We post redacted copies of any legal notices we receive (including original notices, counter notices or retractions) at <https://github.com/github/dmca>. We will not publicly publish your personal contact information; we will remove personal information (except for usernames in URLs) before publishing notices. We will not, however, redact any other information from your notice unless you specifically ask us to. Here are some examples of a published [notice](https://github.com/github/dmca/blob/master/2014/2014-05-28-Delicious-Brains.md) and [counter notice](https://github.com/github/dmca/blob/master/2014/2014-05-01-Pushwoosh-SDK-counternotice.md) for you to see what they look like. When we remove content, we will post a link to the related notice in its place.
 
 Please also note that, although we will not publicly publish unredacted notices, we may provide a complete unredacted copy of any notices we receive directly to any party whose rights would be affected by it.
 
-### F. Repeated Infringement
+### E. Repeated Infringement
 
 It is the policy of GitHub, in appropriate circumstances and in its sole discretion, to disable and terminate the accounts of users who may infringe upon the copyrights or other intellectual property rights of GitHub or others.
 
-### G. Submitting Notices
+### F. Submitting Notices
```

Comment:
@I-Cat Listen people.. seems i need a hacker to figure out who is doing this to me because the best i could do was delete a bunch of thing just to make it stop doing what ever its doing.

Anyone wabt to make a quick buck?
- And yes i do cary a grudge so feeding them gheir own poison then informing the authorities would actually the most just course od action![image](https://user-images.githubusercontent.com/33124419/120117533-588fb400-c196-11eb-86f6-4ddc274f5cc0.jpeg)